rfcs/0167-maintainer-account.md

```diff
@@ -0,0 +1,260 @@
+---
+feature: nixpkgs Maintainers Requirements and Expectations
+start-date: 2023-12-19
+author: Adam C. Stephens
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
```

Comment:
```suggestion
shepherd-team: @Janik-Haag, @RaitoBezarius, @JulienMalka 
shepherd-leader: @Janik-Haag
```

Reply:
@adamcstephens Can you please merge this change?